packages/patternfly-4/react-core/src/components/Title/Title.tsx

```diff
@@ -21,7 +21,7 @@ export interface TitleProps extends Omit<React.HTMLProps<HTMLHeadingElement>, 's
   /** Additional classes added to the Title */
   className?: string;
   /** the heading level to use */
-  headingLevel?: TitleLevel;
+  headingLevel?: TitleLevel | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
Can't you use something like `TitleLevel | keyof TitleLevel`? A slight bit cleaner.

Reply:
I did try that, unfortunately, it does not work. We could create an additional interface as it was before but that felt unnecessary. I think the main thing is that the intellisense works and developers don't need to pull in an entire module just to set a well-known string value.